### PR TITLE
[MM-16232] Update admin definition for push notifications

### PR DESCRIPTION
**Pull request by migbot**
#### Summary
Show ID loaded option only if feature is licensed. I'm using `IDLoadedPushNotifications` as the license feature key. Let me know if that needs to change.

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-16232

### CODE REVIEW

**Comment by enahum** [comment]
@esethna QA needs to test on a server that implements the E20 feature which is not available on this SpinWick, I guess it should be on a test server once all the PR's are merged

File: components/admin_console/admin_definition.jsx [hunk 1]
```diff
@@ -1851,8 +1851,36 @@ const AdminDefinition = {
...
+                        help_text_default: '**Generic description with only sender name** - Includes only the name of the person who sent the message in push notifications, with no information about channel name or message contents.\n **Generic description with sender and channel names** - Includes the name of the person who sent the message and the channel it was sent in, but not the message contents.\n **Full message content sent in the notification payload** -  Includes the message contents in the push notification payload that is relayed through Apple\'s Push Notification Service (APNS) or Google\'s Firebase Cloud Messaging (FCM). It is **highly recommended** this option only be used with an "https" protocol to encrypt the connection and protect confidential information sent in messages.\n **Full message content fetched from the server on receipt** - The notification payload relayed through APNS or FCM contains no message content, instead it contains a unique message ID used to fetch message content from the server when a push notification is received by a device. If the server cannot be reached, a generic notification will be displayed.',
```
Comment (saturninoabril) [inline components/admin_console/admin_definition.jsx]: I see it simple having two schemas (with and without id loaded for push contents). But it's an additional overhead to translation.  Why not combine into one schema only?

File: components/admin_console/admin_definition.jsx [hunk 2]
```diff
@@ -1851,8 +1851,36 @@ const AdminDefinition = {
...
+                        help_text_default: '**Generic description with only sender name** - Includes only the name of the person who sent the message in push notifications, with no information about channel name or message contents.\n **Generic description with sender and channel names** - Includes the name of the person who sent the message and the channel it was sent in, but not the message contents.\n **Full message content sent in the notification payload** - Includes the message contents in the push notification payload that is relayed through Apple\'s Push Notification Service (APNS) or Google\'s Firebase Cloud Messaging (FCM). It is **highly recommended** this option only be used with an "https" protocol to encrypt the connection and protect confidential information sent in messages.\n **Full message content fetched from the server on receipt** - The notification payload relayed through APNS or FCM contains no message content, instead it contains a unique message ID used to fetch message content from the server when a push notification is received by a device. If the server cannot be reached, a generic notification will be displayed.',
```
Comment (saturninoabril) [inline components/admin_console/admin_definition.jsx]: There's unnecessary space here as I commented at https://github.com/mattermost/mattermost-webapp/pull/4262#discussion_r347798629.  Would be nice to fix it.

File: components/admin_console/admin_definition.jsx [hunk 3]
```diff
@@ -1851,8 +1851,36 @@ const AdminDefinition = {
...
+                        help_text_default: '**Generic description with only sender name** - Includes only the name of the person who sent the message in push notifications, with no information about channel name or message contents.\n **Generic description with sender and channel names** - Includes the name of the person who sent the message and the channel it was sent in, but not the message contents.\n **Full message content sent in the notification payload** - Includes the message contents in the push notification payload that is relayed through Apple\'s Push Notification Service (APNS) or Google\'s Firebase Cloud Messaging (FCM). It is **highly recommended** this option only be used with an "https" protocol to encrypt the connection and protect confidential information sent in messages.',
```
Comment (saturninoabril) [inline components/admin_console/admin_definition.jsx]: There's unnecessary space here as I commented at https://github.com/mattermost/mattermost-webapp/pull/4262#discussion_r347798629.  ~~Would be nice to fix it.~~ Update: done fix with my pushed commit

File: components/admin_console/admin_definition.jsx (hunk 1 above)
Comment (enahum) [reply components/admin_console/admin_definition.jsx]: How would you combine the help text into one schema? We looked for a way but found nothing

Comment (saturninoabril) [reply components/admin_console/admin_definition.jsx]: Oh right. I thought that would be as straightforward as in showing/hiding options.  Thanks for looking.

**Pull request merged**